html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email">Email</label>
+      <div>
+        <input type="text" name="email" size="25" required placeholder="Email">
```

Comment:
This `input` needs an `id` to match with the `for` attribute on its `label`

Reply:
do you mean a separate `id` element?